src/libs/actions/User.js

```diff
@@ -53,7 +53,13 @@ function changePasswordAndNavigate(oldPassword, password) {
         .then((response) => {
             if (response.jsonCode !== 200) {
                 const error = lodashGet(response, 'message', 'Unable to change password. Please try again.');
```

Comment:
Please translate this error. Move this to translation files and then use it here. You can get the Spanish translation on slack. Thanks.

Reply:
@parasharrajat I request in slack again today is there any person you know so I will tag that developer. I can translate from google but some time the translation are not correct.
Any suggestion please

Reply:
Let's wait for Monday.